workflow/forms.py

```diff
@@ -1113,16 +1114,51 @@ def __init__(self, *args, **kwargs):
                 'readonly'] = "readonly"
 
 
+class SiteProfileQuickEntryForm(forms.ModelForm):
+    """
+    SiteProfile Quick Entry Form
+    """
+    map = forms.CharField()
+
+    class Meta:
+        model = SiteProfile
+        fields = ['name', 'type', 'longitude', 'latitude']
```

Comment:
Can we remove the longitude and latitude fields from Location modal? The user will not have this information handy when they are entering in a site for the first time. We can enable it when we implement the Map widget.